.github/workflows/using-action.yml

```diff
@@ -0,0 +1,71 @@
+# Note: make sure to use the same version of cypress-io/github-action
+# in all jobs, otherwise the last version wins I believe
+name: Using Cypress GH Action
+on: [push, pull_request]
```

Comment:
@bahmutov 

Maybe filter branch on push? https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

```
on:
  push:
    branches:    
      - master         # Push events on master branch
  pull_request:
      - '*'
```

Reply:
yeah, this is an excellent idea, especially since we open pull requests every time and don't need push